ProcessLib/HT/CreateHTProcess.cpp

```diff
@@ -36,17 +37,42 @@ std::unique_ptr<Process> createHTProcess(
 
     DBUG("Create HTProcess.");
 
+    auto const staggered_scheme =
+        //! \ogs_file_param{prj__processes__process__HT__coupling_scheme}
+        config.getConfigParameterOptional<std::string>("coupling_scheme");
+    const bool use_monolithic_scheme =
+        (staggered_scheme && (*staggered_scheme == "staggered")) ? false : true;
```

Comment:
The `&&` and `==` already return a boolean; no need to have the triple operator:
```c++
const bool smth = (a == b) && !c;
```

Reply:
Changed by using clang-tidy